.github/workflows/r_nightly.yml

```diff
@@ -65,54 +68,58 @@ jobs:
           fi
           echo $PREFIX
 
-          archery crossbow download-artifacts -f r-nightly-packages -t binaries --skip-pattern-validation $PREFIX
+          archery crossbow download-artifacts -f r-nightly-packages -t binaries $PREFIX

```

Comment:
How is `$PREFIX` ever not empty?

Reply:
It is set job-wide [here](https://github.com/apache/arrow/blob/decb9a27e397569df58f260cd3f890a01282fbcc/.github/workflows/r_nightly.yml#L37), it might be better for readability to move that to the step itself.